### ISSUE TITLE
The charts are not rendering after bundling

### ISSUE THREAD

**Issue by rantoinne** [nihgwu/react-native-pie#15]
The charts are not rendering after bundling and in order to see changes I have to turn ON my toggle inspector.
Idea: Can we also implement each section as a clickable type, for Ex: clicking on Red section triggers an event and doing so on Green triggers another..?
![screenshot_1540389589](https://user-images.githubusercontent.com/32746587/47436132-84b74d00-d7c3-11e8-93b7-107ff5ecc65c.png)

**Comment by rantoinne** [comment]
![screenshot_1540536821](https://user-images.githubusercontent.com/32746587/47549659-5a2bd800-d91a-11e8-8b45-f8e12249f15d.png)
 I see this when opening Toggle inspector mode in emulator. This screen and https://github.com/nihgwu/react-native-pie/issues/15#issue-373499757 <- this are same.

**Comment by SnehalAgrawal** [comment]
@rantoinne Did you get any solution?

**Comment by rantoinne** [comment]
> @rantoinne Did you get any solution?

Yes @SnehalAgrawal  I instead used react-native-svg which is the core dependency for this library..had to study the structure of it..but got my solution..

But the above issue still exists in this library

**Comment by rafaelmaeuer** [comment]
I have the same issue on Android after upgrading from RN 0.56 to 0.57.

![Bildschirmfoto 2019-06-14 um 21 32 17](https://user-images.githubusercontent.com/6418459/59534478-b7e35380-8eee-11e9-8fbf-793bf0bbb69e.png)
It looks like this in emulator and on a real device with debug an release builds.

![Bildschirmfoto 2019-06-14 um 21 36 12](https://user-images.githubusercontent.com/6418459/59534480-b9148080-8eee-11e9-9196-9599f7b800a3.png)
It looks like this when enabling the inspector mode on emulator in debug build.

**Comment by cakesoft-ravi** [comment]
@rafaelmaeuer please try and use react-native-svg or you can have a look at this  [https://www.npmjs.com/package/react-native-svgwheel-menu]

**Comment by SnehalAgrawal** [comment]
@rantoinne thanks I'll look into it but for now, I have used react-native-chart-kit. Here is an example for it https://aboutreact.com/react-native-chart-kit/
